packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -12,6 +19,8 @@ export interface MenuButtonState {
   context: MenuButtonContext;
 }
 
+export type MenuButtonTokens = SubmenuTokens & SubmenuItemTokens & ContextualMenuTokens & IButtonTokens;
```

Comment:
I think if you define this as

```
export interface MenuButtonTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {}
```

you should be able to customize the text in menu button with

```
const StyledMenuButton = MenuButton.customize({
    button: {
      borderRadius: 4,
      backgroundColor: '#0095ff',
      borderWidth: 0,
      color: 'yellow',
      variant: 'heroSemibold',
      fontFamily: 'Georgia',
    },
    contextualMenu: { backgroundColor: 'red' },
  });
```

which will render like so:

![image](https://user-images.githubusercontent.com/22876140/126661781-0352bd74-1f12-42e3-a55b-804de387ca27.png)

Reply:
Yeah, I shouldn't use textTokens inside buttom slot. Thank you :)